api/main_endpoints/routes/Messages.js

```diff
@@ -0,0 +1,101 @@
+const {
+  UNAUTHORIZED,
+  FORBIDDEN,
+  SERVER_ERROR
+} = require('../../util/constants').STATUS_CODES;
+const express = require('express');
+const router = express.Router();
+const bodyParser = require('body-parser');
+const User = require('../models/User.js');
+
+router.use(bodyParser.json());
+
+const clients = {};
+
+const writeMessage = ((roomId, message) => {
+  if (clients[roomId]) {
+    clients[roomId].forEach(client => client.res.write(`data: ${JSON.stringify(message)}\n\n`));
+  }
+});
+
+router.post('/send', async (req, res) => {
+
+  const {apiKey, message, id} = req.body;
+
+  let apiKeyFound = false;
+
+  if(!apiKey || !message || !id){
+    res.sendStatus(FORBIDDEN);
+    return;
+  }
```

Comment:
lets do this, leave a nice response. does it work?
```suggestion
  const required = [
    {value: apiKey, title: 'API Key', },
    {value: message, title: 'Message', },
    {value: id, title: 'Room ID', },
  ];
  const missingValue = required.find(({value}) => !value);
  if (missingValue){
    res.status(BAD_REQUEST).send(`You must specify a ${missingValue.title}`);
    return;
  }
```